runtime/include/j9cfg.h.ftl

```diff
@@ -44,7 +44,7 @@ extern "C" {
 #define JAVA_SPEC_VERSION ${uma.spec.properties.JAVA_SPEC_VERSION.value}
 
 #if defined(J9VM_JAVA9_BUILD)
-/* J9VM_JAVA9_BUILD is already defined */
+#  error J9VM_JAVA9_BUILD is already defined
```

Comment:
Why the spaces? `#  error`

Reply:
Any response to this @keithc-ca ?

Reply:
The spacing aligns `error` with `define` below.

Reply:
which is only indented because you indented it =)

Please don't play those indent games in the source.  It inevitably leads to future changes needing to modify more lines to handle the different spacing requirements making for harder reviews.

I'm not going to ask for it to be changed this time but will request that it not be done in the future